mllib/pom.xml

```diff
@@ -156,19 +156,6 @@
     </dependency>
 
   </dependencies>
-  <profiles>
```

Comment:
That's cool if we don't need any LGPL shims included in the build to use the end-user's OpenBLAS, MKL. Yes, as you do, leave it in the parent POM as 'deprecated'